### PR TITLE
MAINTAINERS.md: Added Nico and Alex as remote/storage owners.

### PR DESCRIPTION
**Pull request by bwplotka**
Would love to propose @npazosmendez and @alexgreenbank to help us own remote storage (especially write). 

Nico & Alex are not yet Prometheus maintainer, but I think that’s fine.

### CODE REVIEW

**Review by beorn7 — APPROVED** [review]
Maybe remove @tomwilkie from the list of remote storage maintainers, now that we are on it? I think it is unlikely that he will respond to any PRs in this area.

@bwplotka : Once you have merged this, don't forget to make @alexgreenbank and @npazosmendez members of the Prometheus GH org (which will give them push rights).

**Review by npazosmendez — APPROVED** [review]

**Comment by beorn7** [comment]
Actually, both are already members (because they are v3 release coordinators).

**Review by cstyan — APPROVED** [review]

**Comment by bwplotka** [comment]
> Maybe remove @tomwilkie from the list of remote storage maintainers, now that we are on it? I think it is unlikely that he will respond to any PRs in this area.

Sure, will ask Tom first & do in separate PR (:

**Review by roidelapluie — APPROVED** [review]

**Review by alexgreenbank — APPROVED** [review]
Gulp!

**Pull request merged**
